Getting-Started.md

```diff
@@ -1,3 +1,191 @@
+## Getting started
+
+SirModred relies on ElasticSearch, Kibiter and MySQL/MariaDB. The current versions used are:
+- ElasticSearch 6.1.0
+- Kibiter 6.1.4
+- MySQL/MariaDB (5.7.24/10.0)
+There are 3 options to get started with SirMordred:
```

Comment:
Hi @abhiandthetruth 
Can we have this in a new line, probably you can list and reference the contents (if you feel it is needed)?

Reply:
On it!

Reply:
Is setting up dev environment with pycharm the fourth type @vchrombie? I kinda feel like it is the fourth way one can get started.

Reply:
Not really, there are only three types.
- source code (elasticsearch, kibiter, mysql/mariadb and all the components from source code)
- source code and docker (components from source code and elasticsearch, kibiter, mysql/mariadb from docker)
- docker (elasticsearch, kibiter, mysql/mariadb and components from docker)

Setting up dev env in PyCharm, comes to play when you have grimoirelab components running from source code as we really consider about elasticsearch, kibiter and mysql/mariadb.

I think I am not wrong with my assumption, maybe @valeriocos can approve it.

Reply:
yes @vchrombie , there are only 3 types (maybe the source code option isn't really useful, now that docker containers are everywhere I guess really few people would go for this option). @abhiandthetruth @vchrombie what do you think if we reshape the structure of the geting-started.md? For instance, we could have the following sections:
- Getting started (same content that we have now, plus the content of the section `only docker`
  - getting the containers (content from `source and docker`). In this section we comment about the docker-compose with/without searchguard
  - cloning the repos  (content from `Setting up a Pycharm dev environment`). In this section we comment about forking/cloning the repos and setting up the origin/upstream repos. Maybe, we could include a link to a script that performs the operations above automatically.
  - setting up pycharm (content from `Setting up a Pycharm dev environment`). This section should explain how to set up pycharm with the details about the project interpreter and project structure)
  - execution (content from `Setting up a Pycharm dev environment`). This section should show how to execute micro mordred/sirmordred. Maybe we could add a link to a video that shows an example to collect, enrich and show data from one or more git repositories). Otherwise also some screeshots could work.
- Troubleshooting (same content we have now)
- How to. This section should explain the basic concepts of Elasticsearch and Kibiter using a set of Q&A. For instance, how to log in, how to create a visualization, how to create a dashboard, what is an index pattern, what is an index. In the future, this section could become another md file.

Please share your thoughts, thanks :)

Reply:
Perfect @valeriocos 
This structure would make the instructions more neat and clear.

I just have some minor additions to the above plan.

>   * cloning the repos  (content from `Setting up a Pycharm dev environment`). In this section we comment about forking/cloning the repos and setting up the origin/upstream repos. Maybe, we could include a link to a script that performs the operations above automatically.

I had a [script](https://gist.github.com/vchrombie/18cc5f36fe5c934067addf44a487ead9) for the same purpose. I have written it as per my convenience on my machine. But, we can improve it.
 
>   * execution (content from `Setting up a Pycharm dev environment`). This section should show how to execute micro mordred/sirmordred. Maybe we could add a link to a video that shows an example to collect, enrich and show data from one or more git repositories). Otherwise also some screeshots could work.

This is also a good idea. I am up for it.

Reply:
> I had a script for the same purpose. I have written it as per my convenience on my machine. But, we can improve it.

Perfect! 

> This is also a good idea. I am up for it.

Cool!

@abhiandthetruth if you are OK with the new structure, please decide how you prefer to proceed. We can merge this PR and then start working on the new structure, or you can update it (we will add the missing content in the next iterations).

Thanks

Reply:
@valeriocos @vchrombie Sorry for the late reply, was working on some other additions. I think since the script is made by @vchrombie it will be better if he does the restructuring. WDYT?

Reply:
Maybe, a gist could be ok for the moment (a similar approach has been already used for groupsio and twitter (https://github.com/chaoss/grimoirelab-sirmordred#groupsio, https://github.com/chaoss/grimoirelab-sirmordred#twitter).

Reply:
Definitely a gist would be best!. @vchrombie Shall I proceed. You can provide the gist link in this thread.

Reply:
Thanks @abhiandthetruth.
I will modify the script and send a PR adding the link.